kornia/geometry/conversions.py

```diff
@@ -135,6 +136,25 @@ def convert_affinematrix_to_homography(A: torch.Tensor) -> torch.Tensor:
     return H
 
 
+def convert_affinematrix_to_homography3d(A: torch.Tensor) -> torch.Tensor:
```

Comment:
I think this one can be reused from the previous one, only needs a better naming. Suggestions ? @ducha-aiki @lferraz

Reply:
I will put the implementation into ```_convert_affinematrix_to_homography_impl```. Those functions will be left with shape validation only.